test/functional/feature_coinstatsindex.py

```
@@ -75,6 +76,9 @@ def block_sanity_check(self, block_info, prev_height):
             block_info['new_outputs_ex_coinbase'] + block_info['coinbase'] + block_info['unspendable']
         )
 
+    def sync_index_node(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `sync_index_node`

Add return type annotation: `None`

(ANN201)